### DISCUSSION TITLE
Any plan to open GitHub copilot API?

### DISCUSSION THREAD

**Discussion by Mini256** [community/community#39048] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Do we have any plans to make the copilot API publicly available?

**Comment by anderson-pixel** [comment, marked as answer]
unfortunately not, as of current github hasn't verbally expressed anything about making it public, the only for you to have access to copilot is by paying or utilizing their free trial which you can rescind at any time. albeit you can get it for free if github verifies you as a student or as an open source contributor.

**Comment by jfullstackdev** [comment]
this is the answer

![image](https://user-images.githubusercontent.com/47092464/204072273-4fe101a6-7a82-442d-b18e-54a0fded2f5a.png)
